### PR TITLE
fix(ci): api service schema generation

### PR DESCRIPTION
**Pull request by pshomov**
# ...

Server-side features cannot be evaluated at build time.

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by arnorhs — APPROVED** [review]

**Review by arnorhs — APPROVED** [review]
This has got something to do with the env variables having different values when they are imported?

**Comment by pshomov** [comment]
> This has got something to do with the env variables having different values when they are imported?

No, just lazy evaluating the configuration info since it is affected by the server-side features.

File: apps/api/src/app/environments/environment.ts [hunk 1]
```diff
@@ -1,333 +1,341 @@
...
+          username: process.env.PAYMENT_SCHEDULE_USER,
+          password: process.env.PAYMENT_SCHEDULE_PASSWORD,
+        },
+        islykill: {
+          cert: process.env.ISLYKILL_CERT,
+          passphrase: process.env.ISLYKILL_SERVICE_PASSPHRASE,
+          basePath: process.env.ISLYKILL_SERVICE_BASEPATH,
+        },
+      }
+    : {
```
Comment (darrikonn) [inline apps/api/src/app/environments/environment.ts:163]: Having this huge ternary makes it hard to find where the dev config vs. prod config begins/ends.

My proposal would be to rather keep the same format as before with two variables devConfig and prodConfig and return the previous ternary at the bottom ; makes it easier to read.

**Review by darrikonn — CHANGES_REQUESTED** [review]

File: apps/api/src/app/environments/environment.ts (hunk 1 above)
Comment (pshomov) [reply apps/api/src/app/environments/environment.ts]: 👍🏻

**Review by darrikonn — APPROVED** [review]

**Pull request merged**